test/main.test.js

```diff
@@ -23,6 +23,7 @@ window.wire.env = {
   BACKEND_REST: 'http://localhost',
   BACKEND_WS: 'wss://localhost',
   FEATURE: {
+    ALLOWED_FILE_UPLOAD_EXTENSIONS: ['*'],
```

Comment:
I think for tests the default should be restrictive and should be more relaxed in specific test cases with some mocking.

E.g.:
```
spyOn<{getConfig: () => TypeUtil.RecursivePartial<Configuration>}>(Config, 'getConfig').and.returnValue({
      BACKEND_NAME: 'Test',
      FEATURE: {
        ENABLE_ACCOUNT_REGISTRATION: true,
      },
    });
```

Reply:
Very good point, I will address this in a later PR.